projects/demo-cypress/src/support/component-index.html

```diff
@@ -12,6 +12,11 @@
         />
         <title>Components App</title>
     </head>
+    <script>
+        if (global === undefined) {
+            var global = window;
```

Comment:
Why do we need this? I believe we can use `globalThis` in 4.0 since we bumped browser support.

Reply:
fixed